server/routes/v1/rating.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { verifyUser } from '<middlewares>/verifyUser';
+import { create, update } from '<controllers>/rating';
+import { validateCreateRating, validateUpdateRating } from '<validations>/rating';
+
+const router = express.Router();
+
+router.post('/', verifyUser, validateCreateRating, create);
```

Comment:
I think you should use the checkIsVerified middleware so that only verified users can rate an article as discussed on friday.